common/lib/xmodule/xmodule/modulestore/tests/test_split_modulestore.py

```diff
@@ -624,6 +624,15 @@ def test_get_courses(self, _from_json):
         self.assertEqual(course.edited_by, "testassist@edx.org")
         self.assertDictEqual(course.grade_cutoffs, {"Pass": 0.45})
 
+        course_index = modulestore().get_course_index_info(course.id)
+        new_draft = modulestore().create_course(
+            'testX', 'rerun_2.0', 'run_q2', 1, BRANCH_NAME_DRAFT,
+            versions_dict=course_index['versions'])
```

Comment:
super nit: follow the python guidelines for long lines: https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines